.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest ways to improve Pants
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. E.g., I'm always frustrated when [...]
```

Comment:
Let's frame it in a more positive light, especially because we can't guarantee we'll implement their request. It's about making their experience even better, rather than their experience being bad to begin with.

```suggestion
A clear and concise description of what the problem is. E.g., My life would be better if [...]
```